spec/jobs/get_usps_proofing_results_job_spec.rb

```diff
@@ -444,8 +435,9 @@
           let(:request_delay_ms) { 750 }
 
           it 'adds a delay between requests to USPS' do
+            enrollment_records = InPersonEnrollment.where(id: pending_enrollments.map(&:id))
             allow(InPersonEnrollment).to receive(:needs_usps_status_check).
-              and_return(pending_enrollments)
+              and_return(enrollment_records)
```

Comment:
ditto, is there a way to factory actual records that fit these conditions?

Reply:
Same here, a refactoring of the spec file would be good in the future.